packages/wordcount/README.md

```diff
@@ -27,3 +27,5 @@ count accepts three parameters:
 ```JS
 import { count } from '@wordpress/wordcount';
 const numberOfWords = count( 'Words to count', 'words', {} )
+
+<br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>
```

Comment:
This does not display correctly, as the prior code block is not terminated.

Reply:
There should only be three back ticks to end the block. It does seem to display correctly in the preview though, which is unexpected to me.

Reply:
@aduth thanks for your guidance